### DISCUSSION TITLE
ArgoCD app is missing a resource in web UI

### DISCUSSION THREAD

**Discussion by alxivanov** [argoproj/argo-cd#11860] [Q&A]
**The UI**
![image](https://user-images.githubusercontent.com/89388397/210058200-0d4b78ed-f08a-4749-91be-0dc4049e5164.png)

When syncronizing:
![image](https://user-images.githubusercontent.com/89388397/210059630-4bdc724e-0be3-4692-a337-c7bf454a9448.png)

**Repo**
base:
├── DeploymentConfig.yaml
├── ImageStream.yaml
├── kustomization.yaml
└── Service.yaml
...
stage:
├── DeploymentConfig.yaml
├── ImageStream.yaml
├── ImageStreamTag.yaml
├── kustomization.yaml
└── Service.yaml

base/kustomization.yaml:
```
---
resources:
 - DeploymentConfig.yaml
 - ImageStream.yaml
 - Service.yaml
```

stage/kustomization.yaml:
```
---
bases:
- ../base

namePrefix: stage-

resources:
 - ImageStreamTag.yaml

patchesStrategicMerge: 
  - Service.yaml 
  - DeploymentConfig.yaml
  - ImageStream.yaml
  - ImageStreamTag.yaml
```

I've tried moving ImageStreamTag to base and using PrefixSuffixTransformer to add a suffix, but it didn't work. What am I missing? Or how can I adjust my setup so that the ImageStreamTag resource appears in the web UI overview.

P.S. syncing works fine, the only issue is that the ImageStreamTag resource is missing from the application overview.

**Comment by jannfis** [comment]
Is the ImageStreamTag resource actually created on the cluster?

If yes, could you please post its `metadata` section?

**Reply by alxivanov** [reply]
Hi @jannfis, yes - the resource gets created on the cluster once Openshift establishes that the ImageStream image reference is OK. After that, it is able to get updates via sync. Here is its metadata section:
```
  labels:
    app.kubernetes.io/instance: test-app-stage
  name: app:stage
  namespace: project
```
